homeassistant/requirements.py

```diff
@@ -124,22 +122,16 @@ async def async_process_requirements(
             if pkg_util.is_installed(req):
                 continue
 
+            def _install(hass: HomeAssistant, req: str, kwargs: Dict) -> bool:
+                """Install requirement."""
+                return pkg_util.install_package(req, **kwargs)
+
             ret = await hass.async_add_executor_job(_install, hass, req, kwargs)
```

Comment:
Can be inlined with functools.partial, but at least let's drop `hass` arg.
```suggestion
            def _install(req: str, kwargs: Dict) -> bool:
                """Install requirement."""
                return pkg_util.install_package(req, **kwargs)

            ret = await hass.async_add_executor_job(_install, req, kwargs)
```